app/views/declaration/summary/related_documents.scala.html

```diff
@@ -18,12 +18,12 @@
 @import models.requests.JourneyRequest
 @import services.DocumentType
 @import uk.gov.hmrc.govukfrontend.views.html.components._
-@import views.html.components.gds.changeLink
+@import views.html.components.gds.link
 @import views.html.components.gds.summary_list
 
 @this(
         govukTable: GovukTable,
-        changeLink: changeLink,
+        changeLink: link,
```

Comment:
What is the difference with this?

Those components are used on the summary pages

Reply:
I think the summary page uses "Actions" to create the change links Patryk.    The name change is because this is used to create links for "Remove" as well as "Change"